public/lib/helpers/timeline_helper.js

```diff
@@ -50,15 +78,39 @@ define(function (require) {
      */
     TimelineHelper.prototype.pluckLabel = function (hit, params, notify) {
       let field;
-      if (params.labelFieldSequence) { // in kibi, we have the path property of a field
-        field = kibiUtils.getValuesAtPath(hit._source, params.labelFieldSequence);
-      } else {
-        field = _.get(hit._source, params.labelField);
-      }
-      if (field && (!_.isArray(field) || field.length)) {
-        return field;
+      let value;
+
+      // in kibi, we have the path property of a field
+      if (params.labelFieldSequence || this.isMultiField(hit, params.labelField)) {
```

Comment:
I think you can remove `isMultiField` completely and take care of the "multiField case of kibi" within the `else` block (there is nothing to change).
This way, you can remove line 89.